packages/frint-react/package.json

```diff
@@ -28,6 +28,8 @@
     "frint"
   ],
   "dependencies": {
+    "@types/react": "^15.0.24",
```

Comment:
can we treat them as peerDependencies? like asking developers to install them themselves?

even `react` and `react-dom` are not installed by this package directly.